### PR TITLE
version update and typos

### PR DESCRIPTION
**Pull request by rbaumgar**
- update version for task, pipeline, and pipelinerun for tekton
- some typos in the readme

### CODE REVIEW

**Comment by piyush-garg** [comment]
@rbaumgar can you please squash the commits. Also i did not get the name changes done in the README

**Comment by rbaumgar** [comment]
sorry, I don't know how I can help.

File: README.md [hunk 1]
```diff
@@ -327,54 +327,56 @@ frontend and backend image resource to the correct url with your namespace name
...
 
 A `PipelineRun` is how you can start a pipeline and tie it to the persistentVolumeClaim and params that should be used for this specific invocation.
-
-Lets start a pipeline to build and deploy backend application using `tkn`:
+ 
+Let's start a pipeline to build and deploy the backend application using `tkn`:
 
 ```bash
 $ tkn pipeline start build-and-deploy \
+    --prefix-name build-deploy-api-pipelinerun \
```
Comment (piyush-garg) [inline README.md]: why this change?

File: README.md [hunk 2]
```diff
@@ -327,54 +327,56 @@ frontend and backend image resource to the correct url with your namespace name
...
     -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/01_pipeline/03_persistent_volume_claim.yaml \
     -p deployment-name=pipelines-vote-api \
     -p git-url=https://github.com/openshift/pipelines-vote-api.git \
     -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \
     --use-param-defaults
 
 
-Pipelinerun started: build-and-deploy-run-z2rz8
+Pipelinerun started: vote-api-t9tpq
```
Comment (piyush-garg) [inline README.md]: similarly here why this change?

File: README.md (hunk 1 above)
Comment (rbaumgar) [reply README.md]: have a more meaningful name for the run. Better than the generic one. Otherwise you don't know which object is built and deployed.

File: README.md (hunk 2 above)
Comment (rbaumgar) [reply README.md]: have a more meaningful name for the run. Better than the generic one. Otherwise you don't know which object is built and deployed.

**Comment by sudhishmk** [comment]
You may also have to update api version in ResourceTemplate spec session of 02_template to "v1" as well.

**Comment by rbaumgar** [comment]
@sudhishmk done

**Review by sudhishmk — CHANGES_REQUESTED** [review]
Thanks, it would be better if you could merge all 4 commits to a single commit (can use "git rebase").

**Comment by rbaumgar** [comment]
@sudhishmk: sorry I am not a git expert.

$ git rebase
Current branch master is up to date.

**Comment by sudhishmk** [comment]
> @sudhishmk: sorry I am not a git expert.
> 
> $ git rebase Current branch master is up to date.

You need to use git rebase in interactive mode, a quick search on how merge multiple commits should help.

**Comment by khrm** [comment]
I was trying to push to your branch and it closed the PR. Let's recreate the PR with your commit.

**Comment by khrm** [comment]
OK. So I opened this PR with squashed commits: https://github.com/openshift/pipelines-tutorial/pull/171/commits
